src/main/java/edu/kis/powp/jobs2d/drivers/transformators/TransformationsDriverDecorator.java

```diff
@@ -0,0 +1,23 @@
+package edu.kis.powp.jobs2d.drivers.transformators;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class TransformationsDriverDecorator implements Job2dDriver
```

Comment:
Why this class is called TransformationsDriverDecorator. I cannot see any transformation responsibility.